src/wireviz/Harness.py

```diff
@@ -113,9 +113,17 @@ def create_graph(self):
                                  f'{connector.name}:p{loop[1]}{loop_side}:{loop_dir}')
 
         for _, cable in self.cables.items():
-            awg_fmt = f' ({awg_equiv(cable.gauge)} AWG)' if cable.gauge_unit == 'mm\u00B2' and cable.show_equiv else ''
+
+            if cable.show_equiv:
+                if cable.gauge_unit =='mm\u00B2':
+                    awg_fmt = f' ({awg_equiv(cable.gauge)} AWG)'
+                else:
```

Comment:
Should this be `elif cable.gauge_unit.upper() == 'AWG' or cable.gauge_unit == ''` or something similar to avoid converting 'bananas' to AWG by assuming 'bananas' equals mm2?

Reply:
The 'AWG' and 'mm' constant strings should also be defined only once in the helper file and then use identifiers in the rest of the code.

Reply:
👍  especially the Unicode escaping is a pain. Will you include the fix in this PR? should be easy enough

Reply:
yeah, that was fixed in 625b21e. And i agree about the multiple definitions, but that's a separate refactor job IMO

Reply:
Up to you. Let me know when you're ready to merge this one.

Reply:
If you're fine with the string casing fix for now i'd say go for merge

Reply:
@n42 I'm sorry for pointing out the issue you already had fixed, but I could not see your fix at that time. Github have been slow and somewhat unstable today, so I guess my view was not fully updated.